code/datums/ftl_ships/player_weapons.dm

```diff
@@ -0,0 +1,13 @@
+/datum/player_ship_attack
+	var/cname = "Player Ship Attack"
```

Comment:
i dont know i copied it from the other datum thingy for enemy ships. I thought it was a code name, so I could have a name var later for actual naming stuff